### ISSUE TITLE
[icon request] help, alert, info with no circle

### ISSUE THREAD

**Issue by zoharma** [lucide-icons/lucide#1858]
### Icon name

help, alert, info

### Use cases

We have help, alert and info in circles and other instances, but in a circle button it doesn't look good. In this case would be cleaner to have the icons without any artefacts.

### Design ideas
Options with bigger or smaller circle.

![help-1](https://github.com/lucide-icons/lucide/assets/10050573/503ecb4b-2080-4f9c-bf6e-5e82f5a1e253)
![help](https://github.com/lucide-icons/lucide/assets/10050573/03e2daeb-dc50-46b3-b9f9-3b0914a07d97)

![alert-1](https://github.com/lucide-icons/lucide/assets/10050573/50c01ef8-3a6a-4c1b-ad5b-aac182098d68)
![alert](https://github.com/lucide-icons/lucide/assets/10050573/243ad98f-ff95-488a-b968-ddc6f9bafa68)

![info-1](https://github.com/lucide-icons/lucide/assets/10050573/5e54dc6a-0d4b-4751-9c37-8d98d721aec2)
![info](https://github.com/lucide-icons/lucide/assets/10050573/7328d87a-a067-4567-9b87-c382187aff7d)

SVGs:
![help-1](https://github.com/lucide-icons/lucide/assets/10050573/bef59c8a-ef96-43c7-9ea6-4f9e4aa116db)
![help](https://github.com/lucide-icons/lucide/assets/10050573/10b43f82-a1c9-4f92-bc19-2872ba4aaf34)

![alert-1](https://github.com/lucide-icons/lucide/assets/10050573/3d09b28d-de36-44d5-92be-0c0ec0282c03)
![alert](https://github.com/lucide-icons/lucide/assets/10050573/41305def-13fa-4b08-8518-5d598e012faa)

![info-1](https://github.com/lucide-icons/lucide/assets/10050573/d9ff02ec-fc4b-4dbf-9ad9-040304e16a19)
![info](https://github.com/lucide-icons/lucide/assets/10050573/d80ace04-e928-4279-b8f8-28a094cecf71)

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)
- [X] I have searched existing icons to make sure it does not already exist and I didn't find any.
- [X] I am not requesting a brand logo and the art is not protected by copyright.
- [X] I am not requesting an icon that includes religious, political imagery or hate symbols.
- [X] I have provided appropriate use cases for the icon(s) requested.

**Comment by chessurisme** [comment]
Proposing these icons:
`question-mark`, `exclamation mark`, and `letter-i-small`

![image](https://github.com/lucide-icons/lucide/assets/137697011/3e006fbe-4dd4-47f5-88d0-3d30a635486b)
![image](https://github.com/lucide-icons/lucide/assets/137697011/557ea2ab-f18b-4f09-aba9-b9cc2e8cf134)
![image](https://github.com/lucide-icons/lucide/assets/137697011/8deba71c-b513-49bb-b878-e17313057176)

**Comment by karsa-mistmere** [comment]
Yeah, they definitely need to be larger to somewhat approximate our ideal visual weight, ideally the same size as our letter icons.

There's an issue with a new question mark somewhere, and I have about two dozen icons dealing with our question marks in my draft, I think these are my latest versions for a large set of these:
![image](https://github.com/lucide-icons/lucide/assets/17746067/59817442-c75e-4ed4-8bdf-eaefc8c2ed60)

**Comment by chessurisme** [comment]
Thanks @zoharma for pointing me that. I guess they need more larger version. Also @karsa-mistmere your icons look pretty decent. I like the down ones.

**Comment by karsa-mistmere** [comment]
This is the one I was thinking of: https://github.com/feathericons/feather/issues/550
As referenced in #119.

No matter which route we go, the icons should line up perfectly with our reference letter icons in height.

**Comment by ericfennis** [comment]
@zoharma Looks really nice!
Hmm maybe 4px is not that bad in 100% scale. See https://lucide.dev/icons/arrow-down-to-dot

**Comment by karsa-mistmere** [comment]
> @zoharma Looks really nice! Hmm maybe 4px is not that bad in 100% scale. See https://lucide.dev/icons/arrow-down-to-dot

Naah, 2px is far too large, 1px is quite blurry at 1x, so I think we'd better stick with what we currently have:
![image](https://github.com/lucide-icons/lucide/assets/17746067/d32866fa-b3e3-4e8e-886a-455b54c62346)

**Comment by alamenai** [comment]
Any updates reg this one?

**Comment by waterplea** [comment]
I hope they are coming, would love to be able to use them with background like that:
![image](https://github.com/user-attachments/assets/6a246861-43f4-4b8e-8750-39d9ad703951)

**Comment by jamiemlaw** [comment]
Tried to independently come up with a design for this, and I just ended up with something functionally equivalent to @karsa-mistmere's first design, which I quite like:

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M11+20h.01%22+/%3E%3Cpath+d=%22M7+7q1.5-3+5-3a5+4+0+0+1+5+4c0+4-6+4-6+8%22+/%3E&name=circle-help&base=circle-help"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTEgMjBoLjAxIiAvPgogIDxwYXRoIGQ9Ik03IDdxMS41LTMgNS0zYTUgNCAwIDAgMSA1IDRjMCA0LTYgNC02IDgiIC8+Cjwvc3ZnPg==.svg"/><br/>Open lucide studio</a>

And this is my attempted tracing of @colebemis's icon. I like the fact that it uses circular arcs, but it feels somewhat thin.

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?name=image&base=image&value=%3Cpath+d=%22M12+15c0-3+4-3+4-7a4+4+0+0+0-7.874-1%22+/%3E%3Cpath+d=%22M12+20h.01%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTIgMTVjMC0zIDQtMyA0LTdhNCA0IDAgMCAwLTcuODc0LTEiIC8+CiAgPHBhdGggZD0iTTEyIDIwaC4wMSIgLz4KPC9zdmc+.svg"/><br/>Open lucide studio</a>
